Source/Icebreaker/Content/styles.css

```diff
@@ -28,172 +41,125 @@
         padding-top: 21px;
     }
 
-.bold-header {
-    font-family: 'Segoe UI';
-}
-
 body {
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
     width: 100%;
     height: 100%;
-    margin: 0;
+    margin: 0rem;
     flex-direction: column;
 }
 
-/*Dark Theme rules*/
-.theme-dark .bold-header {
+.bold-header {
     text-align: left;
     font-weight: bold;
     font-size: 18px;
-    color: #FFFFFF;
     letter-spacing: normal;
```

Comment:
Is this needed?

Reply:
Yes - and this was actually coming from the following: https://officedev.github.io/msteams-ui-components/csstest.html